server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -1,12 +1,7 @@
 package views.admin.programs;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-import static j2html.TagCreator.a;
-import static j2html.TagCreator.div;
-import static j2html.TagCreator.each;
-import static j2html.TagCreator.h1;
-import static j2html.TagCreator.h2;
-import static j2html.TagCreator.p;
+import static j2html.TagCreator.*;
```

Comment:
issue (blocking): no wildcard import, looks like this was maybe done automatically by your editor?

Reply:
IDE is adding this. I have changed it back.